manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: lazygamereviews.KEmulator
+PackageVersion: 1.0.3
+PackageLocale: en-US
+Publisher: lazygamereviews
+PackageName: KEmulator

```

Comment:
```suggestion
PublisherUrl: https://archive.org/details/@lazygamereviews
PackageName: KEmulator
PackageUrl: https://archive.org/details/KEmulator103
```

Reply:
I believe it is not a violation

Reply:
> Usually, Archive.org would count as a 3rd party site, but here it appears to be the publisher's only upload location. I do not know if this violates Archive.org's TOS, and if so this package might need to be removed.

"Access to the Archive’s Collections is provided at no cost to you and is granted for scholarship and research purposes only."

Reply:
@stephengillie / @denelon - The more I think about this, should the Waived-Validation-Domain be removed in light of the ToS for archive.org?

Reply:
> > Usually, Archive.org would count as a 3rd party site, but here it appears to be the publisher's only upload location. I do not know if this violates Archive.org's TOS, and if so this package might need to be removed.
> 
> "Access to the Archive’s Collections is provided at no cost to you and is granted for scholarship and research purposes only."
> 
> https://archive.org/about/terms.php

Thanks for finding this.